templates/devenv.yaml.tpl

```diff
@@ -1,3 +1,4 @@
+{{- $_ := stencil.ApplyTemplate "skipIfNotService" -}}
```

Comment:
I think we might've actually had a reason for this... do you recall @george-e-shaw-iv?

Reply:
We wanted to be able to allow libraries to run e2e tests

Reply:
so we no longer need https://outreach-io.atlassian.net/browse/DT-2176?

Reply:
Thats up for y'all to decide